.changeset/nine-pets-join.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sveltejs/kit': minor
+---
+
+feat: warn when usage of page options in `.svelte` files is detected
```

Comment:
should also mention missing `slot` in layout or make this changeset more generic to cover both cases